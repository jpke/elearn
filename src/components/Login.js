import React from 'react';

const Login = (props) => {
  return (
    <div className="quizInProgress">
      <h3 className="authTitle">Login</h3>
      <form onSubmit={props.logIn} id="authRegister">
        <input type="email" id="email" className="auth" placeholder="email" />
        <input type="password" id="password" className="auth" placeholder="password" />
        <button type="submit">Login</button>
      </form>
    </div>
  )
};

export default Login;
