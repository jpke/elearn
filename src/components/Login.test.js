/* eslint-disable no-undef */
/* eslint-disable import/no-named-as-default */

import React from 'react';
import renderer from 'react-test-renderer';

import Login from './Login';

it('renders correctly', () => {
  const tree = renderer.create(
    <Login logIn={jest.fn()} demo={jest.fn()}/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
})
