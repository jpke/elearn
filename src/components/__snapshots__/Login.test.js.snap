exports[`test renders correctly 1`] = `
<div
  className="loginContainer">
  <h3
    className="title">
    Login
  </h3>
  <form
    id="authForm"
    onSubmit={[Function]}>
    <input
      className="auth"
      id="email"
      placeholder="email"
      type="email" />
    <input
      className="auth"
      id="password"
      placeholder="password"
      type="password" />
    <button
      className="authButton"
      type="submit">
      Login
    </button>
  </form>
  <button
    className="authButton"
    onClick={[Function]}>
    Demo
  </button>
</div>
`;
